payroll/models/hr_salary_rule.py

```diff
@@ -202,10 +202,17 @@ def _compute_rule(self, localdict):
                     "result_qty" in localdict and localdict["result_qty"] or 1.0,
                     "result_rate" in localdict and localdict["result_rate"] or 100.0,
                 )
-            except Exception:
+            except Exception as ex:
                 raise UserError(
-                    _("Wrong python code defined for salary rule %s (%s).")
-                    % (self.name, self.code)
+                    _(
+                        """
+                        Wrong python code defined for salary rule %s (%s).
+                        Here is the error received:
+
+                        %s
+                        """
+                    )
+                    % (self.name, self.code, ex)
```

Comment:
```suggestion
                    % (self.name, self.code, repr(ex))
```

and the same for the other

Reply:
Yes it makes sense. Will update the commit and rebase